client/demo_direct_call.sh

```diff
@@ -46,7 +46,7 @@ AMOUNTSHIELD=50000000000
 AMOUNTTRANSFER=40000000000
 
 
-CLIENT="./substratee-client -p ${NPORT} -P ${RPORT}"
+CLIENT="./../bin/substratee-client -p ${NPORT} -P ${RPORT}"
```

Comment:
Some of the scripts use the symbolic link, others have the full path (as I've put in here). Symbolic links unfortunately don't work on Windows (even though I run it in WSL - git cannot handle the symbolic link properly when checking out on a Windows system).